packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -474,15 +593,58 @@ contract ControllerHelper is FlashControllerHelper, IERC721Receiver {
 
             IWPowerPerp(wPowerPerp).transfer(wPowerPerpPool, _amountToPay);
```

Comment:
we have to refund unused ETH here as well, user could sell 1000 long oSQTH and mint 1 oSQTH and sell at 200% CR, and should get most of the proceeds from the sale as a payment of ETH